### PR TITLE
Feature/addlayers

### PR DESCRIPTION
**Pull request by appmen**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -0,0 +1,5 @@
+rootProject.name = 'taf'
+
+include 'api'
+include 'ui'
+include 'core'
```
Comment (Stasnazarenko) [inline settings.gradle]: looks like we discussed that core module will be a separate git sub repo

File: ui/src/main/resources/local.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+url=https://localhost:8080
+
+user=default
+password=1q2w3e
+
+browser=firefox
+timeout=30
```
Comment (Stasnazarenko) [inline ui/src/main/resources/local.properties]: need to discuss the property management,and loading

File: ui/src/main/java/com/learning/configuration/UIConfig.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    public String password() {
+        return config.getProperty("password");
+    }
+    public String browser() {
+        return config.getProperty("browser");
+    }
+    public String timeout() {
+        return config.getProperty("timeout");
+    }
```
Comment (Stasnazarenko) [inline ui/src/main/java/com/learning/configuration/UIConfig.java]: to be discussed

File: ui/src/main/resources/local.properties (hunk 2 above)
Comment (Stasnazarenko) [reply ui/src/main/resources/local.properties]: commom app prop file

File: ui/src/main/java/com/learning/configuration/UIConfig.java (hunk 3 above)
Comment (Stasnazarenko) [reply ui/src/main/java/com/learning/configuration/UIConfig.java]: load automatically all the entire props

File: ui/src/main/resources/local.properties (hunk 2 above)
Comment (appmen) [reply ui/src/main/resources/local.properties]: updated

File: ui/src/main/java/com/learning/configuration/UIConfig.java (hunk 3 above)
Comment (appmen) [reply ui/src/main/java/com/learning/configuration/UIConfig.java]: updated

File: settings.gradle (hunk 1 above)
Comment (appmen) [reply settings.gradle]: updated

**Pull request merged**